src/Task/YamlLint.php

```diff
@@ -54,10 +54,10 @@ public function run(ContextInterface $context): TaskResultInterface
 
         /** @var FilesCollection $files */
         $files = $context->getFiles()->name($extensions);
-        if (count($whitelistPatterns) >= 1) {
+        if (\count($whitelistPatterns) >= 1) {
```

Comment:
Can you change this to `if (\count($whitelistPatterns)) {}`

Reply:
Done